lib/connect-session-sequelize.js

```diff
@@ -25,7 +29,20 @@ module.exports = function SequelizeSessionInit(Store) {
 		}
 		Store.call(this, options);
 
+		var key;
+		this.options = {};
+		for (key in defaultOptions) {
```

Comment:
Ah ok. Yeah as far as I know it's undocumented (don't really know why) but works perfectly :)